demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -30,7 +30,7 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     protected void onStart() {
         super.onStart();
-        player = new PlayerFactory(this, PrioritisedPlayers.prioritiseExoPlayer()).create();
+        player = new PlayerFactory(this, PrioritisedPlayers.prioritiseMediaPlayer()).create();
```

Comment:
Reverted this. We added a comment at the tail end of the last PR to add a switch at some point in the UI.